packages/holocron-dev-server/src/utils/language-packs.js

```diff
@@ -39,31 +39,34 @@ export function loadModuleLanguagePack({
 
   if (ufs.existsSync(languagePackPath)) {
     const fileStats = ufs.statSync(languagePackPath);
-
     if (fileStats.isDirectory()) {
       const languagePack = {};
-      const links = {};
       const scan = [
         ...languagePackFileNames,
         locale,
         moduleName,
       ];
       scan
         .map((fileName) => `${fileName}.json`)
-        .concat('links')
         .map((fileName) => path.join(languagePackPath, fileName))
         .filter((filePath) => ufs.existsSync(filePath))
         .forEach((filePath) => {
-          if (filePath.endsWith('links')) {
-            scan
-              .map((fileName) => path.join(filePath, `${fileName}.json`))
-              .filter((linksFilePath) => ufs.existsSync(linksFilePath))
-              .forEach((linksFilePath) => Object.assign(links, readJsonFile(linksFilePath)));
-          } else {
-            Object.assign(languagePack, readJsonFile(filePath));
-          }
-        });
-      languagePack.links = links;
+          Object.assign(languagePack, readJsonFile(filePath));
+        }
+        );
+      const directories = ufs.readdirSync(languagePackPath, { withFileTypes: true })
```

Comment:
any directory can now be used for creating the language pack that is hot reloaded